containers/Investment/RevolvingPool/View/PoolOverview.tsx

```diff
@@ -1,53 +1,192 @@
+              <TableRow>
+                <TableCell scope="row">Current Reserve</TableCell>
+                <TableCell style={{ textAlign: 'end' }}>
+                  {addThousandsSeparators(toPrecision(baseToDisplay((pool.data as PoolDataV3).reserve, 18), 2))} DAI
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell scope="row">Maximum Reserve Amount</TableCell>
+                <TableCell style={{ textAlign: 'end' }}>
+                  {addThousandsSeparators(toPrecision(baseToDisplay((pool.data as PoolDataV3).maxReserve, 18), 2))} DAI
+                </TableCell>
+              </TableRow>
+            </TableBody>
+          </Table>
 
-      <Table>
-        <TableBody>
-          <TableRow>
-            <TableCell scope="row">Current NAV</TableCell>
-            <TableCell style={{ textAlign: 'end' }}>DAI 20,500</TableCell>
-          </TableRow>
-          <TableRow>
-            <TableCell scope="row">Current Reserve</TableCell>
-            <TableCell style={{ textAlign: 'end' }}>DAI 20,500</TableCell>
-          </TableRow>
-        </TableBody>
-      </Table>
-
-      <Heading level="5" margin={{ bottom: 'small' }}>
-        Assets
-      </Heading>
-      <Table>
-        <TableBody>
-          <TableRow>
-            <TableCell scope="row">Outstanding Volume</TableCell>
-            <TableCell style={{ textAlign: 'end' }}>DAI 1,000.510</TableCell>
-          </TableRow>
-          <TableRow>
-            <TableCell scope="row">DROP APR</TableCell>
-            <TableCell style={{ textAlign: 'end' }}>10.50%</TableCell>
-          </TableRow>
-        </TableBody>
-      </Table>
-    </Box>
+          <Heading level="5" margin={{ bottom: 'small' }}>
+            Assets
+          </Heading>
+          <Table>
+            <TableBody>
+              <TableRow>
+                <TableCell scope="row">Outstanding Volume</TableCell>
+                <TableCell style={{ textAlign: 'end' }}>
+                  {addThousandsSeparators(
+                    toPrecision(baseToDisplay((pool.data as PoolDataV3).outstandingVolume, 18), 2)
+                  )}{' '}
+                  DAI
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell scope="row">
+                  <span>
+                    <TokenLogo src={`../../../../static/DROP_final.svg`} />
+                    DROP APR
+                  </span>
+                </TableCell>
+                <TableCell style={{ textAlign: 'end' }}>{toPrecision(feeToInterestRate(dropRate), 2)} %</TableCell>
+              </TableRow>
+            </TableBody>
+          </Table>
+        </Box>
+
+        <BalanceSheetDiagram direction="row">
+          <BalanceSheetDiagramLeft>
+            <BalanceSheetMidLine>&nbsp;</BalanceSheetMidLine>
+          </BalanceSheetDiagramLeft>
+          <BalanceSheetDiagramRight>&nbsp;</BalanceSheetDiagramRight>
+        </BalanceSheetDiagram>
```

Comment:
This is pretty cool 👍 

Did you test it with different screen sizes?

Reply:
Just improved it a bit, it seems to scale with > tablet size screens.

Reply:
Nice!